tutorials/fluidSolidInteraction/beamInCrossFlow/system/functions

```diff
@@ -1,10 +1,10 @@
 functions
-(
```

Comment:
The functions file should be left in the foam extend form (which uses ` (  );`instead of the OpenFOAM form (which uses `{    }`) as this is what the conversion script expects.

Fixed in a4f1c86f

Reply:
I just realised that this same mistake exists in many of the functions/functions.openfoam files in the solids4foam tutorials. I expect that the solver would give a warning and that's why I didn't notice it.

I see that the functions (foam extend) file should also use `functionObjectLibs` instead of `libs`. Would you be happy to update the other ones? i.e. run `find . -name functions` in the `solids4foam/tutorials` directory and check they all use the correct brackets and functionObjectLibs vs libs. There seems to be only six of them.

Reply:
Hi @philipcardiff, 

https://github.com/solids4foam/solids4foam/pull/58/commits/a4f1c86f6c0da69a36669fca46d4f25567870d58 this one is weird, I had tested this one and it worked with `{ };` (on foam-extend) formats so I left it the way it is !

Reply:
> I just realised that this same mistake exists in many of the functions/functions.openfoam files in the solids4foam tutorials. I expect that the solver would give a warning and that's why I didn't notice it.

Ok I See.
> I see that the functions (foam extend) file should also use `functionObjectLibs` instead of `libs`. Would you be happy to update the other ones? i.e. run `find . -name functions` in the `solids4foam/tutorials` directory and check they all use the correct brackets and functionObjectLibs vs libs. There seems to be only six of them.

Sure, Ill work on this.
@philipcardiff

Reply:
> Hi @philipcardiff,
> 
> [a4f1c86](https://github.com/solids4foam/solids4foam/commit/a4f1c86f6c0da69a36669fca46d4f25567870d58) this one is weird, I had tested this one and it worked with `{ };` (on foam-extend) formats so I left it the way it is !

Hmnn, it would be great if it works with `{ }` then we could remove the need for treating functions differently between versions. Double-check if it works (maybe it does) and then we may have a simple solution. Thanks!

Reply:
Yeah I will double check this and I will keep you posted. 
@philipcardiff